lib/SIL/SILVerifier.cpp

```diff
@@ -1633,6 +1633,12 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
   }
 
   void checkPartialApplyInst(PartialApplyInst *PAI) {
+    if (PAI->getModule().getStage() != SILStage::Raw) {
+      require(!PAI->getFunctionType()->isDifferentiable(),
+              "partial_apply of differentiable funtions is only allowed "
```

Comment:
minor, in case you care to address later:
* `funtions` -> `functions`
* `SWIFT_ENABLE_TENSORFLOW`

Reply:
Darn it, this shouldn’t have been checked in.